jsonnet/jsonnetfile.lock.json

```diff
@@ -59,8 +59,8 @@
           "subdir": ""
         }
       },
-      "version": "73265f5a16de3c1d4b72cdb0f800929c5eb72f03",
-      "sum": "pAo2yL5Ab72oRo6H8gVRkgI3EPqqRDKcEkik+6hhdcA="
+      "version": "0f3097bded073e691bdd799f0edd12d531cc3821",
+      "sum": "0pZPPJCisYuXsVppWe7mJK1e9N6qxMzDDC7uJuAel7M="
```

Comment:
@metalmatze I need to have `jsonnet/vendor` directory changes to have this PR https://github.com/observatorium/deployments/pull/339 in. please see the changes ^^^